web/app/js/components/util/TapUtils.jsx

```diff
@@ -354,7 +354,9 @@ export const tapLink = (d, resourceType, PrefixedLink) => {
   let namespace = d.sourceLabels.namespace;
   let resource = "";
 
-  if (_has(d.sourceLabels, resourceType)) {
+  if (!d.meshed) {
+    return null;
```

Comment:
TIOLI we could return a greyed out, non-clickable Tap icon for resources that aren't tappable

Reply:
will do :sparkles: